### ISSUE TITLE
[bug] set as keyword in reset

### ISSUE THREAD

**Issue by elgs** [highlightjs/highlight.js#2527]
highlight.js 10.0.1:
![image](https://user-images.githubusercontent.com/55955/80931422-97bebc00-8d6e-11ea-9a0b-64e59b83f92f.png)

highlight.js 9:
![image](https://user-images.githubusercontent.com/55955/80931468-dce2ee00-8d6e-11ea-8d15-9efadfbd730a.png)

```html
<pre><code class="javascript">
   class extends LWElement {
      resetName() {
         this.name = 'Leanweb';
      }
   }

</code></pre>
```

And I'm using agate them.

**Comment by yyyc514** [comment]
<img width="495" alt="Screen Shot 2020-05-04 at 8 09 22 AM" src="https://user-images.githubusercontent.com/6473/80964281-94fbaf80-8dde-11ea-8bc7-4e0090893eba.png">

Cant' reproduce.  Please try 10.0.2 and if you're still have a problem a JSFiddle that shows this reproduced would be helpful. 

I'd say perhaps you are using a version without Javascript built in and are getting some other similar language maybe?

**Comment by yyyc514** [comment]
Sorry spoke too soon. This was likely resolved with https://github.com/highlightjs/highlight.js/commit/ba2e65e43f047bd14becc12735d9acbab7c63d4e.

It's still present in 10.0.  You can build from GitHub or it will be fixed in 10.1 release.

**Comment by yyyc514** [comment]
Closing as *resolved already*.

**Comment by yyyc514** [comment]
Resolved in 10.0.3 since technically this was a regression from our last upgrade.

**Comment by elgs** [comment]
Thank you! I confirm it's fixed.
